AndroidAssignment/app/src/main/java/org/sopt/androidassignment/SignInActivity.kt

```diff
@@ -0,0 +1,46 @@
+package org.sopt.androidassignment
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Toast
+import org.sopt.androidassignment.databinding.ActivitySignInBinding
+
+class SignInActivity : AppCompatActivity() {
+
+    private lateinit var binding : ActivitySignInBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivitySignInBinding.inflate(layoutInflater)
+
+        val intent = Intent(this, HomeActivity::class.java)
+        binding.btLogin.setOnClickListener{
+            var str = binding.etId.getText()
+            var pw = binding.etPw.getText()
+            if(!str.isEmpty() && !pw.isEmpty()){
+                Toast.makeText(this, "${str}님 환영합니다.", Toast.LENGTH_SHORT).show()
+                startActivity(intent)
+            }
+            else{
+                Toast.makeText(this, "로그인 실패", Toast.LENGTH_SHORT).show()
+            }
+        }
+
+        val intent2 = Intent(this, SignUpActivity::class.java)
+        binding.btSignin.setOnClickListener{
+            startActivity(intent2)
+        }
+
+        val id = getIntent().getStringExtra("id")
+        val pw = getIntent().getStringExtra("pw")
```

Comment:
마찬가지로 코틀린에서는 프로퍼티라는 기능을 제공하기때문에
```suggestion
              val id = intent.getStringExtra("id")
              val pw = intent.getStringExtra("pw")
```
이렇게만 쓰셔도 됩니다~!!